hubblestack_pulsar/hubblestack_pulsar_win_config.yaml

```diff
@@ -8,6 +8,19 @@ C:\Windows:
     - TakeOwnership
   exclude:
     - C:\Windows\System32
+    - C:\Windows\security\logs\diagnosis.log
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local\lastalive?.dat
+    - C:\Windows\WinSxS\FileMaps
+    - C:\Windows\security\logs
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local
+    - C:\Windows\security\templates\policies
+    - C:\Windows\CCM\Logs
+    - C:\Users\*\AppData\Local\VMware\vpx
```

Comment:
I think it should be under Users section as:
C:\Users: {}
  exclude:
    - C:\Users\*\AppData\Local\VMware\vpx